source/agora/common/EnrollmentManager.d

```diff
@@ -443,4 +515,12 @@ unittest
 
     // test for getEnrollment with removed enrollment
     assert(!man.getEnrollment(utxo_hash2, stored_enroll));
+
+    // test for enrollment block height update
+    assert(!man.getEnrolledHeight(utxo_hash));
+    assert(man.updateEnrolledHeight(utxo_hash, 1));
```

Comment:
Use a value different from `1` (it's too common)

Reply:
I changed it. I like number `9` :)